CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Added a `language` field to the `Account`. Consequently, the `account` and
   `accountList` API responses now include this field. The `insertAccount` and
   `updateAccount` GraphQL API endpoints are also updated to support the field.
+- Added `signInWithNewPassword` GraphQL API for signing in with a new password.
```

Comment:
This is a new API, so should be included in the section "Added".

Reply:
Since the `signIn` has been changed, it should be documented.

Reply:
I'll change it.